tests/Unit/Meilisearch/Builder/CompositeFilterBuilderTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Tests\Unit\Meilisearch\Builder;
+
+use PHPUnit\Framework\TestCase;
+use Setono\SyliusMeilisearchPlugin\Meilisearch\Builder\CompositeFilterBuilder;
+use Setono\SyliusMeilisearchPlugin\Meilisearch\Builder\FilterBuilderInterface;
+
+final class CompositeFilterBuilderTest extends TestCase
+{
+    public function test_it_returns_filters(): void
+    {
+        $brandFilterBuilder = new class() implements FilterBuilderInterface {
+            public function build(array $facets): string|array
+            {
+                $brandQuery = [];
+                /** @var string $size */
+                foreach ($facets['brand'] as $size) {
+                    $brandQuery[] = sprintf('brand = "%s"', $size);
+                }
+
+                return '(' . implode(' OR ', $brandQuery) . ')';
```

Comment:
I suppose we can simplify the implementations of these anonymous classes, they're not relevant in the scope of this test